include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -366,6 +389,24 @@ class MGTransferGlobalCoarsening : public dealii::MGTransferBase<VectorType>
                OutVector &                      dst,
                const MGLevelObject<VectorType> &src) const;
 
+  /**
+   * Restrict fine-mesh field @p src to each multigrid level and store the
+   * result in @p dst.
+   *
```

Comment:
Can you add some of the text I added to the `interpolate` functions above also here, in particular the part on how this differs from the restriction. Also, I suggest to avoid the term `Restrict` as the leading phrase to avoid confusion with the restriction. I think `interpolate` would be better, but I do not recall the exact wording we use in the solution transfer context where this should also be done.

Reply:
This is consistent with:

https://github.com/dealii/dealii/blob/87a5130c56dd5fb8cf7d57b8f130a45d5a874425/include/deal.II/multigrid/mg_transfer_matrix_free.h#L158-L173

Should I change it there too?

Reply:
Yes, that would be great!